### ISSUE TITLE
No perl treads builds available

### ISSUE THREAD

**Issue by ylavoie** [travis-perl/helpers#36]
Although 5.22.0-thr is documented as being available, although we can find it on https://github.com/travis-perl/builds/tree/perl/5.22.0-thr, it fails to work

5.22.0-thr is not installed; attempting download
bzip2: (stdin) is not a bzip2 file.
tar: Child returned status 2
tar: Error is not recoverable: exiting now
...
Setting environment variables from .travis.yml
$ perlbrew use 5.22.0-thr
ERROR: The installation "5.22.0-thr" is unknown.

All 9 latest version show the same behaviour

**Comment by HayoBaan** [comment]
If you look down the build log, you see the travis helper actually _does_ build and install the threaded version of perl (you need to expand the step where the travis helper is called to see it).

Example:
    $ eval $(curl https://travis-perl.github.io/init) --auto
      % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                     Dload  Upload   Total   Spent    Left  Speed
    100   484  100   484    0     0   8122      0 --:--:-- --:--:-- --:--:-- 14235
    Initialized empty Git repository in /tmp/travis-perl-helpers-4DpCQB/.git/
    remote: Counting objects: 49, done.
    remote: Compressing objects: 100% (44/44), done.
    remote: Total 49 (delta 0), reused 25 (delta 0), pack-reused 0
    Unpacking objects: 100% (49/49), done.
    From git://github.com/travis-perl/helpers
     * [new branch]      master     -> origin/master
    Branch master set up to track remote branch master from origin.
    Already on 'master'
    Perl Travis Helpers: heads/master-0-gd2b2e22
    $ build-perl
    Building Perl 5.22-thr .................................... Done.
    $ perl -V
    Summary of my perl5 (revision 5 version 22 subversion 2) configuration:
    ...

**Comment by ylavoie** [comment]
Exactly! What I am seeing is that Travis has to build it instead on downloading it as it does for the non threaded versions. A great time loss. Can we have them pre-built like the others?
